### ISSUE TITLE
The returned ssid is not the one of the network to which my P110 is connected

### ISSUE THREAD

**Issue by goofy2k** [fishbigger/TapoP100#97]
The smart plug returns "VmVyaG9lY2t4X2dsYXM=" as the ssid in the DeviceInfo. This is certainly not the ssid of my network.

I also doubt about the nickname. The program returns "U21hcnQtcGx1ZzU=". This is not the name that I have set in the TAPO app on my Android device.

**Comment by tnmendes** [comment]
This string is in the format base64, so if you decode "U21hcnQtcGx1ZzU=" you will get "Smart-plug5"
https://www.base64decode.org/

**Comment by adrianmmiller** [comment]
yup, as in my example python script, these lines do the job there:

import base64

nickname = base64.b64decode(returnedData['result']['nickname'])
nicknameDecoded = nickname.decode("utf-8")
print ("Plug Name:", nicknameDecoded)

**Comment by goofy2k** [comment]
Thanks guys!  Let's see if I can also see the correct ssid ;-)

**Comment by goofy2k** [comment]
That works OK !  Thanks for hiding my SSID :)))))
